robottelo/hosts.py

```diff
@@ -481,6 +480,22 @@ def subscription_manager_status(self):
     def subscription_manager_list(self):
         return self.execute('subscription-manager list')
 
+    def subscription_manager_environments_set(
+        self,
+        env_names,
+        username=settings.server.admin_username,
+        password=settings.server.admin_password,
+    ):
+        """
+        Reassign the host to the specified content view environments
+        """
+        if env_names is None:
```

Comment:
```suggestion
        assert env_names 
        OR asset isinstance(env_names, str)
```
Here I'd say it would be okay just to assert `env_names` is a truthy-value, or type to be string etc. 
I'm not sure if `ContentHostError` would be the most appropriate to raise, for an issue with environment(s) string passed. But also non-blocking